system-tests/tests/smoke/cre/workflow_don_load_test.go

```diff
@@ -0,0 +1,915 @@
+			for i, donWithMetadata := range setupOutput.donTopology.DonsWithMetadata {
+				containerNames := make([]string, 0, len(donWithMetadata.NodesMetadata))
+				for _, output := range setupOutput.nodeOutput[i].Output.CLNodes {
+					containerNames = append(containerNames, output.Node.ContainerName)
+				}
+				debugDons = append(debugDons, &keystonetypes.DebugDon{
+					NodesMetadata:  donWithMetadata.NodesMetadata,
+					Flags:          donWithMetadata.Flags,
+					ContainerNames: containerNames,
+				})
+			}
+
+			debugInput := keystonetypes.DebugInput{
+				DebugDons:        debugDons,
+				BlockchainOutput: setupOutput.blockchainOutput,
+			}
+			lidebug.PrintTestDebug(t.Name(), testLogger, debugInput)
+		}
+	})
+
+	require.NoError(t, saveFeedAddresses(feedsAddresses))
+
+	//Get ocr key
+	kb := make([]ocr2key.KeyBundle, 0)
+	for _, don := range setupOutput.donTopology.DonsWithMetadata {
+		if flags.HasFlag(don.Flags, keystonetypes.MockCapability) {
+			for i, n := range don.DON.Nodes {
+				if i == 0 {
+					continue // Skip bootstrap nodeker
+				}
+
+				key, err := n.ExportOCR2Keys(n.Ocr2KeyBundleID) //TODO: @george-dorin FIX ME!!!
+				if err == nil {
+					b, err2 := json.Marshal(key)
+					require.NoError(t, err2)
+					kk, err2 := ocr2key.FromEncryptedJSON(b, nodeclient.ChainlinkKeyPassword)
+					require.NoError(t, err2)
+					kb = append(kb, kk)
+				} else {
+					testLogger.Error().Msgf("Could not export OCR2 key: %s", err)
+				}
+
+			}
+		}
+	}
+
+	// Export key bundles so we can import them later in another test, used when crib cluster is already setup and we just want to connect to mocks for a different test
+	require.NoError(t, saveKeyBundles(kb))
+
+	testLogger.Info().Msg("Connecting to mock capabilities...")
+
+	mocksClient := mock_capability.NewMockCapabilityController(testLogger)
+	mockClientsAddress := make([]string, 0)
+	if in.Infra.InfraType == "docker" {
+		mockClientsAddress = []string{"127.0.0.1:13401", "127.0.0.1:13402", "127.0.0.1:13403", "127.0.0.1:13404"}
+	} else {
+		for i, _ := range setupOutput.nodeOutput[1].CLNodes {
+			if i == 0 { // Skip bootstrap node
+				continue
+			}
+			mockClientsAddress = append(mockClientsAddress, fmt.Sprintf("%s-%s-%d-mock.main.stage.cldev.sh:443", in.Infra.CRIB.Namespace, setupOutput.nodeOutput[1].NodeSetName, i-1))
+		}
+	}
+
+	useInsecure := false
+	if in.Infra.InfraType == "docker" {
+		useInsecure = true
+	}
+
+	require.NoError(t, mocksClient.ConnectAll(mockClientsAddress, useInsecure, true)) //Capability don ports
+
+	testLogger.Info().Msg("Hooking into mock executable capabilities")
+
+	receiveChannel := make(chan capabilities.CapabilityRequest, 1000)
+	require.NoError(t, mocksClient.HookExecutables(ctx, receiveChannel))
+
+	labels := map[string]string{
+		"go_test_name": "test1",
+		"branch":       "profile-check",
+		"commit":       "profile-check",
+	}
+
+	_, err = wasp.NewProfile().
+		Add(wasp.NewGenerator(&wasp.Config{
+			CallTimeout: time.Minute * 10,
```

Comment:
that's a huge number, is that done on purpose? that means that a single request can take up 10 minutes before it times out

Reply:
Yes, we want to give the workflow enough time to execute